modules/oe_theme_helper/src/Plugin/PageHeaderMetadata/NodeViewRoutesBase.php

```diff
@@ -110,7 +110,7 @@ protected function getIntroductionMetadata(string $field_name): array {
       // We strip the tags because the ECL component expects only one paragraph
       // of text and the field is using a text format which adds paragraph tags.
       '#type' => 'inline_template',
-      '#template' => '{{ summary|render|striptags("<strong><a><em><svg><use><span>")|raw }}',
+      '#template' => '{{ summary|render|striptags("<strong><a><em><svg><use><span><sub><sup><u><ol><ul><li>")|raw }}',
```

Comment:
The ticket says only <sub><sup><u> to be enabled.

Reply:
I know but the reason we're stripping tags is because the ECL doesn't except multiple p tags while it renders all the other tags allowed in the simple rich text editor. Why shouldn't we enable the ordered and unordered list then?